docs/TROUBLESHOOTING.md

```diff
@@ -139,12 +160,23 @@ If the issue still persists, [create a new issue](https://github.com/openshift/o
   - If a backup or restore is stuck as "In Progress," then it is likely that the backup 
   or restore was interrupted. If this is the case, it cannot resume. 
 
-  - For further details on your backup, run the command `velero backup describe <backup-name>`.
-  And for more details on your restore, run `velero restore describe <backup-name>`.
-
-  - You can delete the backup with the command `oc delete backup <backup-name> -n openshift-adp`,
-  and delete the restore with the command `oc delete restore <restore-name> -n openshift-adp`
+  - For further details on your backup, run the command:
+  ```
+  velero backup describe <backup-name>
```

Comment:
Optionally, include `--details` if you want to see the resource list or volume-level information.

Reply:
that one is on the next line

Reply:
oh.. adding..